sample/src/main/java/com/alfresco/sample/MainViewModel.kt

```diff
@@ -210,12 +213,34 @@ class MainViewModel(private val context: Context) : ViewModel() {
                     facetFormat = "V2"
                 )
 
+                /*
+                *  val where = "(EXISTS(target/file) OR EXISTS(target/folder))"
+        val include = AlfrescoApi.csvQueryParam("path", "allowableOperations")
+        val orderBy = listOf("title ASC")
+                *
+                */
+
+                val where = "(EXISTS(target/file) OR EXISTS(target/folder))"
+                val include = AlfrescoApi.csvQueryParam("path", "allowableOperations")
+                val orderBy = listOf("title ASC")
+
+                val responseFavorite  = favoritesApi.listFavorites(
+                    AlfrescoApi.CURRENT_USER,
+                    0,
+                    25,
+                    orderBy,
+                    where,
+                    include,
+                    null,
+                )
+
 //                val taskList = serviceAPS1.startForm("singlereviewer7-2-23:1:36")
 //                println("data task 11 ==> ${taskList.fields?.first()?.getFieldMapAsList()}")
 
                 val list = searchCall.list?.entries?.map { it.entry } ?: emptyList()
                 results.value = list
 
+                println("Response favorite $responseFavorite")
```

Comment:
can we remove the print statement?

Reply:
It's just a sample class and added to print response as we don't have UI for this here